optimize/client/src/components/Dashboards/AddButton/CreateTileModal.tsx

```diff
@@ -34,7 +34,7 @@ export default function CreateTileModal({close, confirm}: CreateTileModalProps)
   useEffect(() => {
     (async () => {
       const collection = getCollection(pathname);
-      const availableReports = await loadReports(collection);
+      const availableReports = await loadEntities(collection);
```

Comment:
Since we are not filtering for reports here, this is currently listing other entities.